profile/README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to moja global 👋
+
+![moja-global-banner](../assets/moja_global_banner.jpg)
+
+Moja global’s mission is to support ambitious climate action by developing pioneering, open-source software – including the groundbreaking FLINT software – to help users accurately and affordably estimate greenhouse gas emissions and removals from forestry, agriculture and other land uses (AFOLU).
```

Comment:
'groundbreaking' may come across as too strong.

Reply:
I agree with Shloka!
I think it's better to stay humble. :)

Reply:
Fixed! Thanks @chicken-biryani and @shubhamkarande13 :hugs: